db/migrate/20171212042211_create_donations.rb

```diff
@@ -0,0 +1,29 @@
+class CreateDonations < ActiveRecord::Migration
+  def change
+    create_table :donors do |t|
+      t.string :name, null: false
+      t.string :address
+      t.string :email
```

Comment:
QUESTION FOR LISA: Should we add a unique constraint on email instead of name?

Reply:
Don't allow duplicate emails, but allow no email still.

Reply:
Recomend `validates :email, uniqueness: true, allow_nil: true`